src/fsae_firmware/readme.txt

```diff
@@ -0,0 +1,30 @@
+These steps will explain how to deploy and execute new code on the teeny, as well as integrate it with the Jetson.
+
+To upload new code to the teensy:
+Prereqs:
+	Arduino IDE (https://www.arduino.cc/en/Main/Software)
+	Teensy Loader (https://www.pjrc.com/teensy/loader.html)
+
+1. Clone repo (git clone https://github.com/Pitt-RAS/fsae_racecar_2020.git)
+2. cd /PATH/TO/REPO
+3. git checkout jank (switch to jank branch where Noah's poorly updated firmware is kept)
```

Comment:
After this lands, do we still need to change branches?

Reply:
No. It should just be the master branch. I removed this file and replaced it with a README for better formatting and readability.